components/webdriver_server/lib.rs

```diff
@@ -473,6 +473,13 @@ impl Handler {
         &mut self,
         parameters: &NewSessionParameters,
     ) -> WebDriverResult<WebDriverResponse> {
+        if let Ok(value) = env::var("DELAY_AFTER_ACCEPT") {
```

Comment:
The placement of this code is a bit awkward, but I wasn't able to see the output in the terminal if it printed before the webdriver server was actually active.

Reply:
It seems to be a bit hacky as well. Is it possible to add it as a pref instead?